preload/internal/scts/types.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package preload holds code for adding batches of certificates to CT logs.
```

Comment:
Binaries take their name from their folder, so `get`'ing or `install`'ing `./...` was going to create two mystery `main` binaries :) And in general Go doesn't really do `main/` folders.